cmd/main.go

```diff
@@ -114,10 +113,6 @@ func main() {
 		LeaderElectionID:       "7cb6f2e5.ocs.openshift.io",
 		Cache: cache.Options{
 			ByObject: map[client.Object]cache.ByObject{
-				&extv1.CustomResourceDefinition{}: {
-					// only cache storagecluster crd
-					Field: storageclustersSelector,
-				},
```

Comment:
I understand this removal, was a corresponding watch added on this?

Reply:
From what I recall, this is used when we deploy CSI and it will be removed in 4.18. Do we need a watch on this?

Reply:
Until we remove that we need that, is it effecting your feature?

Reply:
Yes, if we specify the CRD to cache here, the operator will only be caching that particular CRD and we cannot have multiple resource names in the field Selector. 
If needed, I can add StorageCluster CRD to how we are watching the CRD (with avail CRD feature). WDYT?